CHANGES.rst

```diff
@@ -27,6 +27,8 @@ New Features
 
 - Add Common Galactic line lists, split Atomic/Ionic list with verified medium info [#1656]
 
+- Add support for nonstandard viewer reference names [#1681]
```

Comment:
It seems kind of... demeaning to reduce such a big infrastructural overhaul to just 7 words 🤣